ticketguardian/auth/auth.py

```diff
@@ -26,3 +26,8 @@ def get_scope(cls, root_type='', root_id=''):
             return Affiliate.retrieve(root_id).scope
         else:
             return cls.retrieve('scope', raw_data=True)
+
+    @classmethod
+    def me(cls):
```

Comment:
I just noticed something that I think we should change slightly and I should have thought of this and included it in the ticket when I was making it. So your current implementation is fine and will work fine but will not work when we want to make an `/auth/me/` call using a different token than ours. With this being a `classmethod` it will default to the keys that are set and will return our information instead of the other token given. So you could do this a few ways but one example would be something like this.
```
@classmethod
    def get_me(cls, **params):
    # ^^ Or a better name you can think of
    """
    GET /auth/me/. Returns a dictionary containing information about the user. Optionally allows you to get another users information using their token.
    Keyword Arguments:
    token (str): A JWT token to use as credentials rather than the user's token.
    """
        return cls.retrieve('me', raw_data=True, instance=cls(**params))
```
Note: I am using `instance` because the sdk makes a new instance of the class to return when using `retrieve` and will use the default keys on that new instance.

Reply:
Thanks for the help! I have just committed these changes to the function.